transport_nantes/mailing_list/views.py

```diff
@@ -226,10 +225,8 @@ def form_valid(self, form):
             mailing_list_token=form.cleaned_data['petition_name'])
         if len(petition) == 0:
             return HttpResponseNotFound("Pétition inconnu")
-        subscription = MailingListEvent.objects.create(
-            user=user,
-            mailing_list=petition[0],
-            event_type=MailingListEvent.EventType.SUBSCRIBE)
+        # Create MailingListEvent that make the user sign the petition
```

Comment:
It doesn't exactly make the user sign, the event is created to account for the petition's signing by the user.
So yeah, it's because the user has signed, we create a MailingListEvent. Your comment kind of make us think the reverse happens somehow. 

Same goes for the previous formulations like : 
```python
# Create MailingListEvent that subscribe the user
# in the  mailing_list
```
You save the user's subscription as a mailing list event. 

I'm quibbling here :wink:

Reply:
Code comments that say exactly what the code does aren't particularly useful:

```
# Add two and two.
y = 2 + 2
```

The event is called `SUBSCRIBE`, it's kind of obvious what that one line is doing, no?

Reply:
Language niggle: distinguish between "make the user sign" and "make sure the user has signed".  They have drastically different meanings. ;-)